nss-day-cohort-30/bangazon-api-whimsical-whooping-cranes
# Katerina controllers

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change
Please delete options that are not relevant.

# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout KaterinaControllers`
1. start BangazonAPI.sln
1. verify...


# Checklist:

TICKET: https://github.com/nss-day-cohort-30/bangazon-api-whimsical-whooping-cranes/issues/6
